build/PlatformInstaller.targets

```diff
@@ -0,0 +1,300 @@
+  </Target>
+
+  <Target Name="_DownloadInstallers">
+    <!-- Download dotnet installers -->
+    <MakeDir Directories="$(_InstallerSource)" />
+    <!--
+      Note: KOREBUILD_DOTNET_FEED_CREDENTIAL is intentionally NOT an MSBuild variable.
+      MSBuild doesn't to the substitution correctly because the string contains %,
+      so we'll let bash do it instead.
+    -->
+    <Exec Command="curl --fail -sSL &quot;$(RuntimeTargzLink)$KOREBUILD_DOTNET_FEED_CREDENTIAL&quot; -o $(_InstallerSource)$(DotnetRuntimeInstallerArchiveName)" />
+  </Target>
+
+  <Target Name="GenerateTargzs" DependsOnTargets="_EnsureInstallerPrerequisites;_DownloadInstallers">
+    <PropertyGroup>
+      <HostingArchiveName>$(HostingInstallerName)-$(PackageVersion)-linux-x64.tar.gz</HostingArchiveName>
+    </PropertyGroup>
+
+    <!-- Clear working directory -->
+    <RemoveDir Directories="$(_WorkRoot)" />
+    <MakeDir Directories="$(_WorkRoot)" />
+
+    <!-- Create layout: Aspnet Hosting Bundle -->
+    <Exec Command="tar -xzf $(SFLinuxArchiveFilePath) -C $(_WorkRoot)" />
+    <Exec Command="tar -xzf $(_InstallerSource)$(DotnetRuntimeInstallerArchiveName) -C $(_WorkRoot)" />
+
+    <!-- Create Aspnet Hosting Bundle tar.gz -->
+    <Exec Command="tar -czf $(_InstallersOutputDir)$(HostingArchiveName) -C $(_WorkRoot) ."/>
+  </Target>
+
+  <Target Name="_BuildDockerImage">
+    <Exec Command="docker build --build-arg USER_ID=%24(id -u) -t docker-image-$(Image) $(Image)" WorkingDirectory="$(_DockerDir)" />
+  </Target>
+
+  <Target Name="_RemoveDockerImage">
+    <Exec Command="docker rmi docker-image-$(Image)" />
+  </Target>
+
+  <Target Name="_GenerateRpm">
+    <!-- Clear working directory -->
+    <RemoveDir Directories="$(_WorkRoot)" />
+    <MakeDir Directories="$(_WorkRoot)" />
+
+    <!-- Create layout: Extract archive if given -->
+    <MakeDir Directories="$(_WorkRoot)package_root\" />
+    <Exec Command="tar -xzf $(SFArchive) -C $(_WorkRoot)package_root\" Condition="'$(SFArchive)'!=''" />
+
+    <!-- Create layout: Place changelog -->
+    <Copy
+      SourceFiles="$(_PackagingDir)changelog"
+      DestinationFiles="$(_WorkRoot)templates/changelog"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+
+    <ItemGroup>
+      <ChangelogItems Include="DATE" Replacement="$([System.DateTime]::UtcNow.ToString(ddd MMM dd yyyy))" />
+      <ChangelogItems Include="MAINTAINER_NAME" Replacement="$(RPMMaintainerName)" />
+      <ChangelogItems Include="MAINTAINER_EMAIL" Replacement="$(RPMMaintainerEmail)" />
+      <ChangelogItems Include="PACKAGE_VERSION" Replacement="$(RPMVersion)" />
+      <ChangelogItems Include="PACKAGE_REVISION" Replacement="$(RPMRevision)" />
+    </ItemGroup>
+
+    <!-- Update Date, Maintainer, Version, Revision and Changelog Message -->
+    <RepoTasks.ReplaceInFile Filename="$(_WorkRoot)templates/changelog" Items="@(ChangelogItems)" />
```

Comment:
(Can be done in cleanup later). Consider using the `GenerateFileFromTemplate` task to find/replace.